qiita_pet/templates/private_studies.html

```diff
@@ -134,6 +137,22 @@ <h4 class="modal-title" id="myModalLabel">Modify Sharing Settings</h4>
             {% end %}
         </tbody>
     </table>
+
+<!--User study Modals-->
+{% for s in user_studies %}
```

Comment:
Yes. How do you propose doing this in javascript? I'm not sure I can even think of a way to do so. This is the easiest way to do it, sonce it makes the modals and shows them as needed.